ui/jobs/jobs.js

```diff
@@ -1563,6 +1564,73 @@ class Bars {
     };
   }
 
+  // (In Progress) (Need Help)
+  setupSch() {
+    let gcd = kUnknownGCD;
+
+    let BioBox = this.addProcBox({
+      id: 'sch-procs-bio',
+      fgColor: 'sch-color-bio',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    let AetherflowBox = this.addProcBox({
+      id: 'sch-procs-aetherflow',
+      fgColor: 'sch-color-aetherflow',
+      scale: gcd,
+      threshold: gcd * 3, // you need at most 3 gcd to cast all 3 stacks out
+    });
+
+    let AetherflowRescourceBox = this.addResourceBox({
+      classList: ['sch-color-aetherflow'],
+    });
+
+    let LucidDreamingBox = this.addProcBox({
+      id: 'sch-procs-luciddreaming',
+      fgColor: 'sch-color-lucid',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    this.jobFuncs.push((jobDetail) => {
+      let aetherflow = jobDetail.aetherflowStacks;
+      AetherflowRescourceBox.innerText = aetherflow;
+      AetherflowBox.threshold = gcd * aetherflow;
+
+      let p = AetherflowRescourceBox.parentNode;
+      if (aetherflow == 3 && AetherflowBox.duration <= 15) {
+        // then to red when under 15 seconds
+        // but you even have 3 aetherflow stacks
+        p.classList.add('need-to-throw');
```

Comment:
What do you think about reusing the `pulse` CSS class instead of a new `need-to-throw`? That is used for BLM when you are going to lose a xenoglossy and it makes the resource box shake around a bit.

Reply:
I've tried this class. I think pulse is not good for this use, because BLM have only 1 xeno to lost so it use gradual change to show the time limit coming. But SCH have 3 stack to overwrite, and different actions can be done. A bright red also alert them to use these stacks.